README.md

```diff
@@ -52,23 +65,25 @@ Exit the server:
 exit
 ```
 
-## Usage
+# Usage-of-myProxy-Website
 
-Go to server URL:
-```
-http://your-server-ip-address:3000
-```
+Sign in using admin password and use provider's API key and secret to setup domains:
 
-You will be prompted to enter your admin password and your domain provider's API Key and Secret, [find out how here](https://github.com/Neilpang/acme.sh/wiki/dnsapi)
+1. Go to server URL: ```http://my\_server\_ip:3000```
+2. Sign in with my\_admin\_password to gain access to admin page
+3. Create domain provider's API Key and Secret, [find out how here](https://github.com/Neilpang/acme.sh/wiki/dnsapi#4-use-godaddycom-domain-api-to-automatically-issue-cert)
+4. Input GD key and GD Secret, enter one at a time
+5. You will now be able to see your domains, click on setup
 
 After your domain is setup, you will be able to generate as many subdomain repository as you want! To do that:
-1. Go to your server url:  `http://your-server-ip-address:3000`
-2. Create a subdomain. Ip and port are optional. You should see a git link that was created for you.
-3. `git clone` the app, then build the app locally. Find out how in the Building Your Local App section below.
-4. When you are done, `git push origin master` and watch your app run in production!
 
+1. Go to your server url using My Proxy button or type url:  `http://my\_server\_ip:3000`
+2. Create a subdomain. Ip and port are optional. You should see a git link `<your fullDomain repo>` that was created for you, copy the git link for use below.
 
 ## Building-Your-Local-App 
+
+Build the app locally. When you are done watch your app run in production:
```

Comment:
Here you are saying `build the app locally`, but below you are telling users to clone a new repo.

There is another part. If you hhave your app built locally, you can dowload the cofig file, add start script, commit your files, add remote url `git remote add prod myproxy@......`.